client/a8c-for-agencies/sections/sites/sites-header-actions/index.tsx

```diff
@@ -23,6 +24,9 @@ export default function SitesHeaderActions( { onWPCOMImport }: Props ) {
 
 	return (
 		<div className="sites-header__actions">
+			{ config.isEnabled( 'a4a-dev-sites' ) && (
+				<AddNewSiteButton showMainButtonLabel={ ! isMobile } devSite />
```

Comment:
**💬 For discussion:** How do we want to handle displaying the button on mobile viewport?

At the moment, simple `+` is being used. The issue is that both buttons display it at the moment:

![Markup on 2024-07-24 at 17:13:14](https://github.com/user-attachments/assets/37872e79-04c2-4ab8-8095-7f0341cb74d4)

Alternatively, we could consider placing the new CTA into existing popover:

![Markup on 2024-07-24 at 17:15:34](https://github.com/user-attachments/assets/3f5ed850-e750-4574-bb44-65810b8bac5d)

But if I recall correctly, there was some discussion that we want to keep the CTA button separate.

Reply:
Related discussion: p1721893821881679/1721834414.546509-slack-C02TCEHP3HA.

Reply:
**Update:** For now (until we get input from the design team) we will be displaying two `+` buttons. I have also created a new task to keep track of the issue: https://github.com/Automattic/dotcom-forge/issues/8458.

→ Marking this discussion as solved.